meilisearch/models/index.py

```
@@ -62,3 +62,63 @@ class EmbedderDistribution(CamelBase):
+
+class UserProvidedEmbedder(CamelBase):
+    source: str = "userProvided"
+    dimensions: int
+    distribution: Optional[EmbedderDistribution] = None
+    binary_quantized: Optional[bool] = None
+
+
+class Embedders(CamelBase):
+    embedders: Dict[
+        str,
+        Union[OpenAiEmbedder, HuggingFaceEmbedder, OllamaEmbedder, RestEmbedder, UserProvidedEmbedder],
```

Ruff (0.11.9)

Undefined name `Union`

(F821)